paddle/fluid/operators/activation_op_npu.cc

```diff
@@ -431,6 +431,63 @@ class ReciprocalGradNPUKernel : public framework::OpKernel<T> {
   }
 };
 
+template <typename DeviceContext, typename T>
+class CosNPUKernel : public framework::OpKernel<T> {
+ public:
+  void Compute(const framework::ExecutionContext& ctx) const override {
+    auto* x = ctx.Input<Tensor>("X");
```

Comment:
specify const identifier to remind

Reply:
prefer to maintain a style consistent with this .cc file